File: java/Encapsulation.java
public class Encapsulation {
    private int x;
    public void setX(int a){
        x=a;
    }

    public int getX(){
        return x;
    }

    public static void main(String[] args) {
        Encapsulation e = new Encapsulation();
        e.setX(100);
        System.out.println(e.getX());
    }
}

File: MUJAHID/c++/Encapsulation.cpp
#include<iostream>
using namespace std;

class A{

    private : int x;

    public:
    void setX(int x){
        this->x=x;
    }

    int getX(){
        return x;
    }

};

int main(){

    A a ;
    a.setX(10);
    cout<<a.getX();

    return 0;
}